test/src/test/java/org/corfudb/benchmark/SMRMapBenchmark.java

```diff
@@ -0,0 +1,207 @@
+package org.corfudb.benchmark;
+
+import com.google.common.reflect.TypeToken;
+
+import java.util.Random;
+import java.util.concurrent.atomic.AtomicInteger;
+import java.util.stream.IntStream;
+
+import org.corfudb.runtime.CorfuRuntime;
+import org.corfudb.runtime.collections.SMRMap;
+import org.corfudb.runtime.exceptions.TransactionAbortedException;
+import org.openjdk.jmh.annotations.Benchmark;
+import org.openjdk.jmh.annotations.Measurement;
+import org.openjdk.jmh.annotations.Threads;
+import org.openjdk.jmh.annotations.Warmup;
+
+/**
+ * A simple benchmark for {@link org.corfudb.runtime.collections.SMRMap} operations.
+ *
+ * Created by zlokhandwala on 5/3/19.
+ */
+public class SMRMapBenchmark extends AbstractCorfuBenchmark {
+
+    /**
+     * State for simple {@link org.corfudb.runtime.collections.SMRMap} benchmarks.
+     * Keeps a single map, and a {@link AtomicInteger} which is used as a counter
+     * to issue new keys.
+     */
+    public static class SMRBenchmarkState extends CorfuSingleNodeBenchmarkState {
+
+        /**
+         * A runtime to use during the iteration.
+         */
+        CorfuRuntime rt;
+
+        /**
+         * A map to use during each iteration.
+         */
+        SMRMap<Integer, Integer> map;
+
+        /**
+         * A counter to be used during each iteration.
+         */
+        AtomicInteger counter;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/app/annymartinez.m/CorfuDB/pullRequest?prid=4651477)